lib/commands/init.js

```diff
@@ -64,5 +65,17 @@ module.exports = function (path = '', local, theme) {
     replace(target(filename), 'repo: \'\'', `repo: '${repo}'`)
   }
 
+  if (plugins) {
+    replace(target(filename), '_plugins_', '_plugin\n  '.repeat(plugins.length))
+    plugins.forEach(plugin => {
+      const major = version[0]
+      const url = plugin.includes('//') ? plugin : `https://cdn.jsdelivr.net/npm/docsify@${major}/lib/plugins/${plugin}.min.js`
```

Comment:
if use `./bin/docsify init docs2 -p 1`, An exception will be thrown here

![image](https://user-images.githubusercontent.com/33931153/116375006-7093b100-a841-11eb-872d-717fc3ae1843.png)

Reply:
I added the enquirer in #117, you can use this here as well.

see https://github.com/enquirer/enquirer#multiselect-prompt

![](https://raw.githubusercontent.com/enquirer/enquirer/master/media/multiselect-prompt.gif)

Reply:
Ideally, we should support specifying a list of plugins as an argument to '--plugins' such that the prompt shows up on supplying an invalid plugin or none.

```sh
docsify init [path] --plugins, -p [...plugins]
```